cmd/tailscale/cli/whois.go

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var whoisCmd = &ffcli.Command{
+	Name:       "whois",
+	ShortUsage: "whois [--json] [ip|ip:port]",
```

Comment:
We accept both IPv4 and IPv6 here, right? Maybe add to long help

Reply:
done